python/kvikio/kvikio/cufile_driver.py

```diff
@@ -10,6 +11,26 @@
 DriverProperties = cufile_driver.DriverProperties
 
 
+def libcufile_version() -> Tuple[int, int]:
+    """Get the libcufile version.
+
+    Returns (0, 0) for cuFile versions prior to v1.8.
+
+    Notes
+    -----
+    This is not the version of the CUDA toolkit. cufile is part of the
+    toolkit but follows its own version scheme.
+
+    Returns
+    -------
+    The version as a tuple (MAJOR, MINOR).
+    """
+    v = cufile_driver.libcufile_version()
+    major = v // 1000
+    minor = (v % 1000) // 10
+    return (major, minor)
```

Comment:
Maybe we can use `divmod` to consolidate these a bit?

```suggestion
    major, minor = divmod(v, 1000)
    minor //= 10
    return (major, minor)
```